src/components/join-flow/country-step.jsx

```diff
@@ -0,0 +1,115 @@
+const bindAll = require('lodash.bindall');
+const classNames = require('classnames');
+const React = require('react');
+const PropTypes = require('prop-types');
+import {Formik} from 'formik';
+const {injectIntl, intlShape} = require('react-intl');
+
+const countryData = require('../../lib/country-data');
+const FormikSelect = require('../../components/formik-forms/formik-select.jsx');
+const JoinFlowStep = require('./join-flow-step.jsx');
+
+require('./join-flow-steps.scss');
+
+/**
+ * Return a list of options to give to select
+ * @param  {object} reactIntl      react-intl, used to localize strings
+ * @param  {string} defaultCountry optional string of default country to put at top of list
+ * @return {object}                ordered set of county options formatted for frc select
+ */
+const getCountryOptions = reactIntl => {
+    const processedCountryData = countryData.processedCountryOptions;
+    processedCountryData.unshift({
+        label: reactIntl.formatMessage({id: 'registration.selectCountry'}),
+        disabled: true,
+        value: 'null'
+    });
+    return processedCountryData;
+};
+
+class CountryStep extends React.Component {
+    constructor (props) {
+        super(props);
+        bindAll(this, [
+            'handleValidSubmit',
+            'validateForm',
+            'validateSelect'
+        ]);
+    }
+    validateSelect (selection) {
+        if (selection === 'null') {
+            return this.props.intl.formatMessage({id: 'form.validationRequired'});
+        }
+        return null;
+    }
+    validateForm () {
+        return {};
```

Comment:
IIRC, In the current we actually validate the country you choose against the list of acceptable ones. Does that belong here or does that happen server side in scratchr2 as part of create user?

Reply:
hm, not sure what you mean about "in the current". you mean, scratchr2 does that? where does it do that?

Reply:
If you're talking about checking the value the user provides against this list, maybe that's something we can do later, before submitting?

Reply:
Yeah. I'm not sure where it is or if we do it, but I got that impression from a conversation we had in a meeting the other day.  How about putting in a TODO to figure this out later?

Reply:
I think that's beyond the scope of the front end -- I think any validation that the country the front end provided is valid belongs in the back end. It would be different if we were allowing the user to type out their own country string.